app/views/compositions/index.html.erb

```diff
@@ -0,0 +1,27 @@
+
+<%= link_to "Create composition", new_composition_path, class: 'btn btn-primary' %>
+
+<div class="container">
+  <div class="card m-5 p-5" style="width: 36rem;">
+  <% @compositions.each do |composition| %>
+    <div class="card-body">
+        <ul>
+          <h4>Memory List</h4>
+          <% composition.memory_list.each do |phrase| %>
+          <ul>
+            <li><%= phrase  %></li>
```

Comment:
Is this a placeholder? I guess we will print all the flashcards in the memory list here right?

this can be left out of the index, as it is in the show